ui_framework/components/menu/menu.js

```diff
@@ -0,0 +1,39 @@
+import React, {
+  PropTypes,
+} from 'react';
+import classNames from 'classnames';
+
+const getClassName = ({ className, contained = false }) =>
+  classNames('kuiMenu', className, {
+    'kuiMenu--contained': contained
+  });
+
+
+const KuiMenu = ({
+  contained,
+  className,
+  children,
+  ...rest
+}) => {
+  return (
+    <ul
+      className={getClassName({
```

Comment:
Ok, as
```
 const classes = classNames('kuiMenu', className, {
    'kuiMenu--contained': contained
  });
```
to avoid name clash with the parameter `className`